src/modules/operate/AccountOperation.ts

```diff
@@ -161,6 +168,38 @@ export class AccountOperation {
     return this;
   }
 
+  public approveLimitOrder(args: CallApproveLimitOrder): AccountOperation {
+    const APPROVE_LIMIT_ORDER = new BigNumber(0);
+    this.addActionArgs(
+      args,
+      {
+        actionType: ActionType.Call,
+        otherAddress: this.contracts.limitOrders.options.address,
+        data: toBytes(
+          APPROVE_LIMIT_ORDER,
+          this.limitOrders.getOrderHash(args.order),
+        ),
+      },
+    );
+    return this;
+  }
+
+  public cancelLimitOrder(args: CallCancelLimitOrder): AccountOperation {
```

Comment:
can you also add ones that override this and approveLimitOrder but take a string order hash instead?

Reply:
Okay changed the structs to take order or orderhash